Sources/SwiftyXPC/XPCDecoder.swift

```diff
@@ -271,7 +271,7 @@ public final class XPCDecoder {
 
                 return XPCFileDescriptor(fileDescriptor: xpc_fd_dup(xpc)) as! T
             } else if #available(macOS 11.0, *), type == FileDescriptor.self {
-                try checkType(xpcType: XPC_TYPE_FD, swiftType: XPCFileDescriptor.self, xpc: xpc)
+                try checkType(xpcType: XPC_TYPE_FD, swiftType: FileDescriptor.self, xpc: xpc)
```

Comment:
I'm not sure if this is a typo or intended

Reply:
That was a typo. Thanks!